dxr/plugins/clang/dxr-index.cpp

```diff
@@ -80,45 +79,57 @@ struct FileInfo {
   std::string realname;
   std::ostringstream info;
   bool interesting;
+  static std::string srcdir;  // the project source directory
```

Comment:
> We probably also have to put IndexConsumer in charge of creating FileInfos, since that's what's interested in most of those vars.

I think that's already true: `IndexConsumer::getFileInfo` seems to be the only creator of `FileInfo`s :-)